features/step_definitions/basic_steps.rb

```diff
@@ -1 +1,16 @@
-# put step definitions in this file
\ No newline at end of file
+Given(/^I visit the homepage$/) do
+  visit '/'
+end
+
+Then(/^I would like to see the text "([^"]*)"$/) do |text|
+  expect(page).to have_content text
+end
+
+Given(/^I visit the amber page$/) do
+  visit '/name/Amber'
+end
```

Comment:
This is not bad, but I would prefer if you could make this step definition more generic and pass in a name as a variable

Reply:
name is now a variable